tests/test_predict.py

```diff
@@ -313,12 +316,39 @@ def test_format_species_probs_too_few_species(self):
 class TestEmbed(unittest.TestCase):
     def test_get_image_features(self):
         classifier = TreeOfLifeClassifier(device='cpu')
-        self.assertEqual(classifier.model_str, 'hf-hub:imageomics/bioclip')
+        self.assertEqual(classifier.model_str, 'hf-hub:imageomics/bioclip-2')
         features = classifier.create_image_features_for_image(EXAMPLE_CAT_IMAGE, normalize=False)
-        self.assertEqual(features.shape, torch.Size([512]))
+        self.assertEqual(features.shape, torch.Size([768]))
 
     def test_get_image_features_pil(self):
         classifier = TreeOfLifeClassifier(device='cpu')
-        self.assertEqual(classifier.model_str, 'hf-hub:imageomics/bioclip')
+        self.assertEqual(classifier.model_str, 'hf-hub:imageomics/bioclip-2')
         features = classifier.create_image_features_for_image(PIL.Image.open(EXAMPLE_CAT_IMAGE), normalize=False)
-        self.assertEqual(features.shape, torch.Size([512]))
+        self.assertEqual(features.shape, torch.Size([768]))
+
+
+class TestEnsureTolSupportedModel(unittest.TestCase):
+    def test_ensure_tol_supported_model_valid(self):
+        # Should not raise for supported models
+        try:
+            ensure_tol_supported_model(BIOCLIP_V1_MODEL_STR)
+            ensure_tol_supported_model(BIOCLIP_V2_MODEL_STR)
+        except Exception as e:
+            self.fail(f"ensure_tol_supported_model raised unexpectedly: {e}")
+
+    def test_ensure_tol_supported_model_invalid(self):
+        with self.assertRaises(ValueError) as cm:
+            ensure_tol_supported_model("hf-hub:some/unsupported-model")
+        self.assertIn("Tree-of-Life predictions are only supported for the following models", str(cm.exception))
```

Comment:
```suggestion
        self.assertIn("TreeOfLife predictions are only supported for the following models", str(cm.exception))
```
Matching my suggested change in the file.